back/engines/free/email_campaigns/config/locales/en.yml

```diff
@@ -410,87 +440,86 @@ en:
-      header_title: 'An idea you follow has a new status'
-      subject: 'The status of an idea you follow has been changed'
-      preheader: 'An idea you follow has a new status'
+      status_change: 'The new status of this input is ''%{status}'''
+      header_message: '%{organizationName} updated the status of the input ''%{input_title}'' on their digital participation platform.'
+      header_title: 'An input you follow has a new status'
+      subject: 'The status of "%{input_title}" has changed'
+      preheader: 'An input you follow has a new status'
     status_change_on_initiative_you_follow:
       status_change: 'The new status of this proposal is ''%{initiativeStatus}'''
       header_message: 'The status of the proposal ''%{initiativeTitle}'' has been updated on the digital participation platform of %{organizationName}.'
       header_title: 'A proposal you follow has a new status'
       subject: 'The status of a proposal you follow has changed'
       preheader: 'A proposal you follow has a new status'
     user_digest:
-      subject: "Your activity on the participation platform of %{organizationName}"
+      subject: "This week on the participation platform of %{organizationName}"
       commented: "%{authorFirstName} commented:"
       preheader: "Weekly overview of %{organizationName}"
       title_your_weekly_report: "Discover what happened last week"
       intro_text: "Here's a summary of what happened on the participation platform of %{organizationName}."
       cta_go_to_the_platform: "Go to the platform"
       title_no_activity_past_week: "No activity in the past week"
-      title_proposals: "Proposals started"
-      proposals_started_text: "Proposals are ideas that need to gain traction and support from others (people like you!) in order to move forward. Below is a list of proposals that have been put forward in the last week. Click on any of them to take a look and offer your support and feedback!"
-      title_successful_proposals: "Proposals that reached the threshold"
-      successful_proposals_text: "These proposals have gotten enough support to move to the next stage! Click on the proposal to learn more about what happens next."
+      successful_proposals_title: "Reached the threshold"
+      successful_proposals_text: "In the past week, these have gotten enough support to move to the next stage."
       today_by_author: "Today by %{author}"
       yesterday_by_author: "Yesterday by %{author}"
       x_days_ago_by_author: "%{x} days ago by %{author}"
-      trending_ideas: "Trending ideas"
-      trending_ideas_text: "Interested in what’s been happening on the platform? Here are the three most trending ideas and what people are saying about them."
+      trending_title: "Trending"
+      trending_text: "Interested in what’s been happening on the platform? Here are the three most trending contributions and what people are saying about them."
```

Comment:
@kogre Just checking, do we want to keep "contributions" (which is also one of the input terms) or do we want the generic term "inputs"?